hnn_core/viz.py

```diff
@@ -233,8 +246,10 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
         The SciPy function :func:`~scipy.signal.decimate` is used, which
         recommends values <13. To achieve higher decimation factors, a list of
         ints can be provided. These are applied successively.
-    color : tuple of float
+    color : tuple of float | str
         RGBA value to use for plotting. By default, 'k' (black)
+    sim_name : str | None
+        Prefix of dipole labels. Used when overlaying multiple simulations.
```

Comment:
could we just call it `label` to adhere to matplotlib naming convention? It could default to `'average'`